src/ocp_postprocess/pull_secret_rename.rs

```diff
@@ -0,0 +1,63 @@
+use crate::{config::ConfigPath, k8s_etcd::InMemoryK8sEtcd};
+use anyhow::{Context, Result};
+use std::{path::Path, sync::Arc};
+
+mod etcd_rename;
+mod filesystem_rename;
+pub(crate) mod utils;
```

Comment:
The utils module is only used by this pull secret module and its children, so `pub(crate)` is not necessary

```suggestion
mod utils;
```